# Define upper left and lower right point of the bounding box
int16 x1
int16 y1
int16 x2
int16 y2
# Define class label
int16 class_id
string class_description
